### DISCUSSION TITLE
What is the Nuxt 3 equivalent of enabling app.config.performance = true?

### DISCUSSION THREAD

**Discussion by KyleBrown-804** [nuxt/nuxt#25779] [Questions]
I have not been able to find any documentation online or examples that show how to enable performance tracing for chrome in Nuxt 3. I can see how if I was using Vue 3 I could enable it via `app.config.performance = true` but I don't see a way to do this for nuxt.

Vue 3 way to enable it: https://vuejs.org/api/application#app-config-performance
SO example: https://stackoverflow.com/a/64503359/14189232

I assume this should be a setting in nuxt.config.ts but cannot find anything on it.

**Comment by AloisSeckar** [comment]
It is a property of Vue app instance, so you can reach it easilly like `useNuxtApp().vueApp.config.performance`

**Reply by KyleBrown-804** [reply]
Gotcha, so in that case for enabling performance tracing I should be able to do something like this in a plugin and enable it globally I suppose? https://nuxt.com/docs/guide/directory-structure/plugins#creating-plugins

Will try this out and see if it works.

**Reply by KyleBrown-804** [reply]
Got it working as a plugin thanks
